project/Build.scala

```diff
@@ -25,7 +25,7 @@ object ScaldingBuild extends Build {
   val avroVersion = "1.7.4"
   val bijectionVersion = "0.8.1"
   val cascadingAvroVersion = "2.1.2"
-  val chillVersion = "0.7.0"
+  val chillVersion = "0.7.1-t1439774596000-77b8a73d7643263249f76ae03c0d3f03db330fc3"
```

Comment:
Yes exactly, this version isn't on maven central, so this won't pass tests.  -- branch hasn't merged in Chill either, and no rls there.